packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1291,19 +1291,53 @@ model = SwarmFormerModel.from_pretrained("${model.id}")
 `,
 ];
 
-export const mlx = (model: ModelData): string[] => [
+const mlx_unknown = (model: ModelData): string[] => [
 	`pip install huggingface_hub hf_transfer
 
 export HF_HUB_ENABLE_HF_TRANSFER=1
 huggingface-cli download --local-dir ${nameWithoutNamespace(model.id)} ${model.id}`,
 ];
 
+const mlxlm = (model: ModelData): string[] => [
+	`pip install --upgrade mlx-lm
+
+mlx_lm.generate --model ${model.id} --prompt "Hello"`,
+];
```

Comment:
An alternative is to use a "real" snippet like the one in the [model cards](https://huggingface.co/mlx-community/Qwen3-0.6B-4bit), but I think a CLI one-liner is enough here.